native/chat/inner-text-message.react.js

```diff
@@ -111,27 +90,11 @@ class InnerTextMessage extends React.PureComponent<Props> {
 }
 
 const styles = StyleSheet.create({
-  darkLinkText: {
-    color: colors.light.link,
-    textDecorationLine: 'underline',
-  },
-  emojiOnlyText: {
```

Comment:
We need to figure out a way to keep the increased-emoji-size logic before we can merge this